### ISSUE TITLE
`Made text edits` message is not very informative

### ISSUE THREAD

**Issue by dbaeumer** [microsoft/vscode#208576]
Testing #208409

![Image](https://github.com/microsoft/vscode/assets/1931590/fb6c2d92-3bce-4d5c-882c-675d3337de35)

In the end everything will be text edits. So not sure how useful this message is. My be we can combine it with the `Accept or Discard` info. Something like

Made one textual changes. Please Accept or Discard them.

**Comment by jrieken** [comment]
related to https://github.com/microsoft/vscode/issues/208586

**Comment by jrieken** [comment]
Agree, it's merely a placeholder and needs tweaking. I like the idea of combining and I would like to but the diff-toggle there

**Comment by jrieken** [comment]
I tweaked the working but we aren't in a position to make this more rich. The message "Made changes" is defined by the data model, the number of hunks is defined by the view model which also does the progressive typing

**Comment by jrieken** [comment]
Closing this as duplicate of https://github.com/microsoft/vscode-copilot/issues/5672. The "Made changes" messages only shows when not other textual reply was provided and we wanna change things so that the extension spells out what it did, like `/fix` already does it today
